css/style.css

```diff
@@ -231,3 +231,97 @@ footer h5 {
 .p3-text p {
   font-style: italic;
 }
+
+.main2 {
```

Comment:
In the design, the two sections under the `main` tag share the width 50% each. How can you do that here? Maybe `grid` would be simpler, to do that?